.github/workflows/test.yaml

```
@@ -119,15 +111,10 @@ jobs:
           args: --release --out dist
 
       - name: Run tests
+        env:
+          RUST_BACKTRACE: 1
         run: |
           git submodule update --init
-          export PATH=$PATH:$HOME/d/protoc/bin
-          export PROTOC=$HOME/d/protoc/bin
-          sudo chown -R $(whoami) $HOME/d/protoc
-          ls -l $HOME/d/protoc/
-          ls -l $HOME/d/protoc/bin
-          pip install datafusion-python --no-index --find-links dist --force-reinstall
-          pip install pytest
-          cargo clean
-          maturin develop
-          RUST_BACKTRACE=1 pytest -v .
+          source venv/bin/activate
+          pip install .
```

Think we'll want to do an editable install here:

```suggestion
          pip install -e . -vv
```

Also added verbose flags so we can catch if anything goes wrong

---

Just pushed with this suggestion. Thank you. I will add you to my fork in a second so you can push changes if needed.